src/job.ts

```diff
@@ -377,6 +396,7 @@ export class Job {
         cp.stdin.write(`set -eo pipefail\n`);
 
         if (this.imageName) {
+            cp.stdin.write(`umask 000\n`); // this allows cache files to be managed on host
```

Comment:
Can you explain to me the implications of setting umask to 000 ?
And is there some way to test how remote runners set the umask to?

Reply:
Nvm... Found out...
![image](https://user-images.githubusercontent.com/973602/114231367-a64c2500-997a-11eb-9fb9-75bd5b0a3ab0.png)
Is there something preventing us from using the default 0002 umask. Like remote runners do ?

Reply:
Scratch that, its the image that determines the umask, so we definently can't override like this.
![image](https://user-images.githubusercontent.com/973602/114231484-ced41f00-997a-11eb-958c-0bb750c622f6.png)

Reply:
```yaml
job:
  image: alpine
  script:
    - umask

test:
  image: debian
  script:
    - umask
```

Reply:
I set umask to be able to delete cache without being root.

Now that files are in /tmp/ that is not that important, so I'll delete this line.

Reply:
Done